frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -104,6 +105,25 @@ export class TemplateValidations {
     return { min, max, isMinInclusive, isMaxInclusive, isValid };
   };
 
+  getAllowedBusses = (): Set<string> => this.getAllowedEnumValues(ValidationJSONPath.BUS);
```

Comment:
does not take into account the name of the rule - e.g. `windows-virtio-bus`,  `windows-disk-bus` which has a meaning with consequences

Reply:
I don't understand what you wanted to say in this comment

Reply:
ok, let's not do that and just enforce the rules